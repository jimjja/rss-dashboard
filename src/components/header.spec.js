import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';

describe('Header component', () => {
  it('should match snapshot', () => {
    const props = {
      siteTitle: 'RSS Dashboard',
    };
    const tree = renderer.create(<Header {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
