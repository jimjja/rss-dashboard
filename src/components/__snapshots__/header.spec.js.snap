// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div
  style={
    Object {
      "background": "rebeccapurple",
      "marginBottom": "1em",
      "padding": "1em .5em",
    }
  }
>
  <h1
    style={
      Object {
        "color": "white",
        "margin": 0,
        "textAlign": "left",
        "textDecoration": "none",
      }
    }
  >
    RSS Dashboard
  </h1>
</div>
`;
